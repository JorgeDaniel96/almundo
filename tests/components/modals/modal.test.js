import React from "react";
import { View } from "react-native";
import Modal from "~/src/components/modals/modal";
import renderer from "react-test-renderer";

it("Modal render is correct with children", () => {
  const tree = renderer.create(<Modal children={<View />} />).toJSON();
  expect(tree).toMatchSnapshot();
});
