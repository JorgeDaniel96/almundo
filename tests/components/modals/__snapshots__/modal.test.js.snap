// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal render is correct with children 1`] = `
<View
  style={
    Array [
      undefined,
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(36, 32, 32, 0.8)",
        "bottom": "0%",
        "elevation": 20,
        "height": 1334,
        "justifyContent": "center",
        "position": "absolute",
        "width": 750,
        "zIndex": 100,
      },
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 5,
        "height": "70%",
        "paddingHorizontal": 15,
        "width": "80%",
      }
    }
  >
    <View
      style={
        Object {
          "height": "100%",
          "justifyContent": "space-around",
          "width": "100%",
        }
      }
    >
      <View />
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "elevation": 10,
          "height": 30,
          "justifyContent": "center",
          "left": 15,
          "opacity": 1,
          "position": "absolute",
          "top": 15,
          "width": 30,
          "zIndex": 10,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Image
        source={
          Object {
            "testUri": "../../../assets/images/iconClose.png",
          }
        }
        style={
          Object {
            "width": "100%",
            "zIndex": 10,
          }
        }
      />
    </View>
  </View>
</View>
`;
